src/components/Projects/WBS/WBSDetail/EditTask/EditTaskModal.jsx

```diff
@@ -253,7 +253,7 @@ const EditTaskModal = props => {
       classification,
     };
 
-    props.updateTask(props.taskId, updatedTask);
+    props.updateTask(props.taskId, updatedTask, hasPermission(role, 'editTask'));
```

Comment:
hasPermission should receive 4 parameters (role, action, roles, userPermissions). The roles is the state that get all roles and permissions of each one in the API, and the userPermissions take the permissions of an individual user (if he had)
check line 630 (for example)